From a9841944b1cd0a6830cee6c31dad8ebe26432fc6 Mon Sep 17 00:00:00 2001
From: sb6596 <shubhamprince111@gmail.com>
Date: Mon, 14 Oct 2019 15:02:06 +0000
Subject: [PATCH 1/4] sepolicy: Label /firmware(/.*)? instead of
 /firmware/image(/.*)?

 * We need the mount point to have a label too.
   Just extend u:object_r:firmware_file:s0 to the parent dir.

Change-Id: Ic989c9d010cd6300c06a28a4f9394687b1453d0c
---
 common/file_contexts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/common/file_contexts b/common/file_contexts
index 76d20e8..30a96a0 100644
--- a/common/file_contexts
+++ b/common/file_contexts
@@ -676,7 +676,7 @@
 /vendor/firmware(/.*)?                                              u:object_r:firmware_file:s0
 /system/etc/firmware(/.*)?                                          u:object_r:firmware_file:s0
 /system/vendor/firmware(/.*)?                                       u:object_r:firmware_file:s0
-/firmware/image(/.*)?                                               u:object_r:firmware_file:s0
+/firmware(/.*)?                                                     u:object_r:firmware_file:s0
 
 /(vendor|system/vendor)/bin/grep                                    u:object_r:vendor_toolbox_exec:s0
 ##################################
-- 
2.7.4

